docs/News.md

```diff
@@ -1,3 +1,49 @@
+    * [High-Grade Serous Ovarian Cancer (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=hgsoc_tcga_gdc) *602 samples*
+    * [Invasive Breast Carcinoma (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=brca_tcga_gdc) *1103 samples*
+    * [Lung Adenocarcinoma (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=luad_tcga_gdc) *571 samples*
+    * [Lung Squamous Cell Carcinoma (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=lusc_tcga_gdc) *503 samples*
+    * [Miscellaneous Neuroepithelial Tumor (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=mnet_tcga_gdc) *184 samples*
+    * [Non-Seminomatous Germ Cell Tumor (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=nsgct_tcga_gdc) *156 samples*
+    * [Pancreatic Adenocarcinoma (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=paad_tcga_gdc) *186 samples*
+    * [Papillary Renal Cell Carcinoma (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=prcc_tcga_gdc) *292 samples*
+    * [Papillary Thyroid Cancer (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=thpa_tcga_gdc) *515 samples*
+    * [Pleural Mesothelioma (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=plmeso_tcga_gdc) *87 samples*
+    * [Prostate Adenocarcinoma (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=prad_tcga_gdc) *501 samples*
+    * [Rectal Adenocarcinoma (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=read_tcga_gdc) *171 samples*
+    * [Renal Clear Cell Carcinoma (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=ccrcc_tcga_gdc) *537 samples*
+    * [Soft Tissue (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=soft_tissue_tcga_gdc) *264 samples*
+    * [Stomach Adenocarcinoma (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=stad_tcga_gdc) *443 samples*
+    * [Thymoma (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=thym_tcga_gdc) *124 samples*
+    * [Uterine Carcinosarcoma/Uterine Malignant Mixed Mullerian Tumor (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=ucs_tcga_gdc) *57 samples*
+    * [Uveal Melanoma (TCGA, GDC)](https://www.cbioportal.org/study/summary?id=um_tcga_gdc) *80 samples*
+    * [Breast (CPTAC, GDC)](https://www.cbioportal.org/study/summary?id=breast_cptac_gdc) *154 samples*
+    * [CNS/Brain (CPTAC, GDC)](https://www.cbioportal.org/study/summary?id=brain_cptac_gdc) *74 samples*
+    * [Colon Adenocarcinoma (CPTAC, GDC)](https://www.cbioportal.org/study/summary?id=coad_cptac_gdc) *109 samples*
+    * [Head and Neck Carcinoma, Other (CPTAC, GDC)](https://www.cbioportal.org/study/summary?id=ohnca_cptac_gdc) *150 samples*
+    * [Lung Adenocarcinoma (CPTAC, GDC)](https://www.cbioportal.org/study/summary?id=luad_cptac_gdc) *180 samples*
+    * [Lung Squamous Cell Carcinoma (CPTAC, GDC)](https://www.cbioportal.org/study/summary?id=lusc_cptac_gdc) *90 samples*
+    * [Ovary/Fallopian Tube (CPTAC, GDC)](https://www.cbioportal.org/study/summary?id=ovary_cptac_gdc) *113 samples*
```

Comment:
```suggestion
    * [Ovarian Cancer (CPTAC, GDC)](https://www.cbioportal.org/study/summary?id=ovary_cptac_gdc) *113 samples*
```

Reply:
Not 100% sure about this one since I am removing `Fallopian Tube`, but this sounds smoother than `Ovary/Fallopian Tube Cancer`. Additionally, none of the existing studies we have on the ovary mention the fallopian tube. Thoughts?

Reply:
That makes sense. Lets have it as `Ovarian Cancer` to be consistent with other studies.